include/ck/tensor_operation/gpu/element/element_wise_operation.hpp

```diff
@@ -172,6 +172,35 @@ struct AddAdd
     }
 };
 
+// C = A * B
+// E = (C + D0) x D1
+struct AddMultiply
+{
+    template <typename E, typename C, typename D0, typename D1>
```

Comment:
Please do like this if only certain datatype are supported

<img width="751" alt="image" src="https://user-images.githubusercontent.com/22615726/207743977-44e1d1ea-7730-4451-83de-2e0e53ce43f0.png">

Reply:
done